thefuck/shells/powershell.py

```diff
@@ -0,0 +1,17 @@
+from .generic import Generic
+
+class Powershell(Generic):
+    def app_alias(self, fuck):
+        return  'function ' + fuck + ' { \n' \
```

Comment:
Also, there are 2 spaces between `return` and `'`. Please remove that one and align it linedown.